### PR TITLE
change project name in header, point home link to start

### PR DESCRIPTION
**Pull request by fordie**
All tests passed

### CODE REVIEW

**Comment by peteanning** [comment]
looking this

File: app/uk/gov/hmrc/offpayroll/views/govuk_wrapper.scala.html [hunk 1]
```diff
@@ -22,13 +22,13 @@
 }
 
 @headerNavLinks = {
-  <li><a id="homeNavHref" href="http://www.google.com"
+  <li><a id="homeNavHref" href="/check-your-employment-status-for-tax/start"
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/views/govuk_wrapper.scala.html]: @routes.InterviewController.back()

File: app/uk/gov/hmrc/offpayroll/views/interview/step_success.scala.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 @()(implicit request: Request[_], messages: Messages, flash: Flash)
 
-@uk.gov.hmrc.offpayroll.views.html.main_template(title = "Hello from off-payroll-frontend", bodyClasses = None) {
+@uk.gov.hmrc.offpayroll.views.html.main_template(title = "Hello from Check your employment status for tax", bodyClasses = None) {
```
Comment (peteanning) [inline app/uk/gov/hmrc/offpayroll/views/interview/step_success.scala.html]: delete this file

**Comment by fordie** [comment]
updated link to use helper. removed unused view

**Comment by fordie** [comment]
All tests passed again

**Pull request merged**